diff --git a/metagpt/provider/openai_api.py b/metagpt/provider/openai_api.py
index 877bd71..65cbe16 100644
--- a/metagpt/provider/openai_api.py
+++ b/metagpt/provider/openai_api.py
@@ -149,6 +149,9 @@ class OpenAILLM(BaseLLM):
             # compatible to openai o1-series
             kwargs["temperature"] = 1
             kwargs.pop("max_tokens")
+        if self.model.startswith("gpt-5"):
+            kwargs["max_completion_tokens"] = kwargs.pop("max_tokens")
+            kwargs.pop("temperature", None)
         if extra_kwargs:
             kwargs.update(extra_kwargs)
         return kwargs
diff --git a/requirements.txt b/requirements.txt
index 39fe0c6..90ef961 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -9,7 +9,7 @@ fire==0.4.0
 typer==0.9.0
 # godot==0.1.1
 # google_api_python_client==2.93.0  # Used by search_engine.py
-lancedb==0.4.0
+lancedb==0.38.0
 loguru==0.6.0
 meilisearch==0.21.0
 numpy~=1.26.4
